charts/jenkins/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: jenkins
 home: https://jenkins.io/
-version: 2.6.2
+version: 2.6.3
```

needs changelog entry too?

```suggestion
version: 2.6.5
```